.github/scripts/sprint_initializr_bump_solver.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# Expects the following environment variables to be set:
+#   $SPRING_INITIALIZR_YAML_FILE_PATH   (Example: "application.yml")
+#   $TIMEFOLD_SOLVER_VERSION            (Example: "1.6.0")
+
+# Temporary file
+temp_file="temp.yml"
+
+# Flag to indicate if the 'timefold-solver' keyword is found
+found_keyword=false
+
+# Read the YAML file line by line
+while IFS= read -r line
```

Comment:
After looking at the `application.yaml` I finally understand why this loop cannot be just a simple `sed` command (all hail the yaml files).